omf/fileio/geoh5.py

```diff
@@ -170,9 +208,22 @@ def from_omf(self, parent=None, **kwargs) -> Data:
 
         return self._entity
 
-    def from_geoh5(self) -> UidModel:
-        """TODO Convert geoh5 entity to omf element."""
-        raise NotImplementedError
+    def from_geoh5(self, **kwargs) -> UidModel:
+        """Convert a geoh5 data to omf element."""
+        with fetch_h5_handle(self.geoh5):
+            kwargs = self.collect_h5_attributes(**kwargs)
+            uid = kwargs.get("uid")
```

Comment:
Just use pop instead of get and you can drop the del statement!

Reply:
Nice, of course. Forgot about that one.